samples/meetings-sidepanel/csharp/SidePanel/ClientApp/package.json

```diff
@@ -20,8 +20,8 @@
     "fluid-framework": "~0.59.0",
     "lodash": "^4.17.21",
     "msteams-ui-components-react": "^0.8.4",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
```

Comment:
Why we have reverted to 16?

Reply:
The project dependencies were not compatible with react 18 so we are using react 16